projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.ts

```diff
@@ -624,6 +627,61 @@ export class NgxLocationPickerComponent implements OnInit, OnDestroy, ControlVal
     this.leafletNotification = null;
   }
 
+  /**
+   * Registers geolocation watchpositon event
+   */
+  private startWatchPosition(): void {
```

Comment:
er is geen manier om tracking aan of af te zetten na het initieel renderen van de component. Ik denk dat dit te beperkend is, dus je zou beter dat benaderen als een methode waarmee je tracking aan en af zet met een bepaalde configuratie, maar waarbij je ook via de initiële configuratie dit automatisch kan triggeren.

Reply:
Hierbij gewoon via ngOnChanges kijken naar value changes op TrackPosition en zo tracking aan of af zetten?

Reply:
dat is ook een manier, maar de readme kan dit dan best wel verduidelijken

Reply:
aangepast